### PR TITLE
Drop type specs in HLTEvF, MuonMonitor and Physics

### PR DESCRIPTION
**Pull request by NiharSaha**
#### PR description:
Drop type specs in DQM/HLTEvF, DQM/MuonMonitor and DQM/Physics python configuration files.

Central DQM migration campaign for drop type spces following
https://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1
and update the safer syntax for existing parameters like: (1) drop type specifications where the original parameter exists. (2) using "clone" instead of "deepcopy" (3) move all parameters inside the clone

#### PR validation:
Tested in CMSSW_12_3_X via runTheMatrix.py -l limited -i all --ibeos

### CODE REVIEW

**Comment by missirol** [comment]
@NiharSaha for `HLTEvF`, I see some type-specifications also in [1,2]. Any reason not to update those?

[1] https://github.com/cms-sw/cmssw/blob/a9f44839b30cab42562af4434e1f01c8b1af8cc6/DQM/HLTEvF/python/HLTTrackingMonitoring_Client_cff.py#L7
[2] https://github.com/cms-sw/cmssw/blob/a9f44839b30cab42562af4434e1f01c8b1af8cc6/DQM/HLTEvF/python/HLTTrackingMonitoring_cff.py#L5

**Comment by NiharSaha** [comment]
@missirol thanks for pointing it out. I think I had missed that. I will commit again to the same branch.

File: DQM/HLTEvF/python/HLTTrackingMonitoring_cff.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 # tracking monitor
 from DQMOffline.Trigger.TrackingMonitoring_cff import *
-iterHLTTracksMonitoringHLT.doProfilesVsLS   = cms.bool(True)
-iterHLTTracksMonitoringHLT.beamSpot = cms.InputTag("hltOnlineBeamSpot")
-pixelTracksMonitoringHLT.beamSpot = cms.InputTag("hltOnlineBeamSpot")
+doProfilesVsLS   = True
+beamSpot = "hltOnlineBeamSpot"
+beamSpot = "hltOnlineBeamSpot"
```
Comment (missirol) [inline DQM/HLTEvF/python/HLTTrackingMonitoring_cff.py:7]:
```suggestion
iterHLTTracksMonitoringHLT.doProfilesVsLS = True
iterHLTTracksMonitoringHLT.beamSpot = "hltOnlineBeamSpot"
pixelTracksMonitoringHLT.beamSpot = "hltOnlineBeamSpot"
```
It should be like this, I think.

**Comment by missirol** [comment]
please test

File: DQM/HLTEvF/python/HLTTrackingMonitoring_cff.py (hunk 1 above)
Comment (NiharSaha) [reply DQM/HLTEvF/python/HLTTrackingMonitoring_cff.py]: Thank you. It was not showing any error though, strange! I have committed your suggestions, hope this is ok to do.

**Comment by missirol** [comment]
+hlt

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**